clicache/integration-test2/Cluster.cs

```diff
@@ -71,7 +73,7 @@ private bool StartLocators()
             for (var i = 0; i < locatorCount_; i++)
             {
                 var locator = new Locator(this, new List<Locator>(),
-                    name_ + "/locator/" + i.ToString(), i == 0);
+                    name_ + "/locator/" + i.ToString(), i == 0, allowAttach_);
```

Comment:
This constructor is starting to get wild. What about a builder patter?

Reply:
I do like the builder pattern. With nearly 30 potential parameters for the start locator command it seems prudent.

Reply:
Done.